components/mdns/mdns.c

```diff
@@ -3831,6 +3877,44 @@ void mdns_parse_packet(mdns_rx_packet_t *packet)
                             _mdns_remove_scheduled_answer(packet->tcpip_if, packet->ip_protocol, type, service);
                         }
                     }
+                    if (service) {
+                        mdns_parsed_record_t *record = malloc(sizeof(mdns_parsed_record_t));
+                        if (!record) {
+                            HOOK_MALLOC_FAILED;
+                            goto clear_rx_packet;
+                        }
+                        record->next = parsed_packet->records;
+                        parsed_packet->records = record;
+                        record->type = MDNS_TYPE_PTR;
```

Comment:
should we also say that the record type is `MDNS_ANSWER`. Not used anywhere, but for the sake of people reading the code?

Reply:
Yes, it's a greate suggestion. Thanks.